apps/ensrainbow/Dockerfile

```diff
@@ -1,10 +1,11 @@
+ARG PNPM_VERSION=9.12.0
 # ensrainbow-data contains an already ingested DB that will be served by the ENSRainbow app
 # Using a prebuilt image for ensrainbow-data (infrequently changes) reduces build time for the ENSRainbow app image
-FROM ghcr.io/namehash/ensnode/ensrainbow-data AS ensrainbow-data
+FROM --platform=linux/arm64 ghcr.io/namehash/ensnode/ensrainbow-data AS ensrainbow-data
```

Comment:
I guess it will work in reverse way: build data on "standard" GitHub architecture (amd64?) and use here with `platform=linux/amd64`.